### PR TITLE
chore: Set Node version with Volta

### PR DESCRIPTION
**Pull request by puskuruk**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,5 +22,8 @@
   "homepage": "https://github.com/kamp-us/monorepo#readme",
   "devDependencies": {
     "turbo": "^1.6.3"
+  },
+  "volta": {
+    "node": "19.2.0"
```
Comment (usirin) [inline package.json:27]: wouldn't it be better to make this the version of LTS?

**Comment by usirin** [comment]
@puskuruk ty so much for the first contribution, really great to see you here!

**Comment by puskuruk** [comment]
> @puskuruk ty so much for the first contribution, really great to see you here!

Thank you! You'll see me more here 💃

File: package.json (hunk 1 above)
Comment (puskuruk) [reply package.json:27]: I think it would better. Thank you for the suggestion. I checked it it looks like `18.14.12` has LTS. I'm updating my pr 🙏

Comment (puskuruk) [reply package.json]: Pr has updated 💃

**Review by usirin — APPROVED** [review]

**Pull request merged**